js/ui/iconGrid.js

```diff
@@ -477,6 +477,13 @@ const IconGrid = new Lang.Class({
         this.saturation = new Shell.GridDesaturateEffect({ factor: 0,
                                                            enabled: false });
         this.actor.add_effect(this.saturation);
+
+        /* Setup the composite mode of the grid */
+        Main.layoutManager.connect('monitors-changed', Lang.bind(this, function() {
```

Comment:
You don't need to wrap `this._updateCompositeMode()` in another function here - you can just use it inside the call to `Lang.bind()`